portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -66,12 +66,33 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8150 | Test Summary: Page views update in real time"
-	@ignore = "true"
 	@priority = "5"
 	test PageViewsUpdateInRealTime {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentPageViews = selenium.getText("ACUsage#USAGE_CURRENT_PAGE_VIEWS");
+
+		ACUtils.launchDXP();
+
+		ACUtils.closeAllSessions();
+
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
 
-		// TODO LRAC-8150 Usage#PageViewsUpdateInRealTime pending implementation
+		ACSettings.goToUsage();
 
+		AssertTextNotEquals(
```

Comment:
Hi @AfonsoHSantos , I would suggest you to compare if the number is increased by 1 (+1) after the new interaction with DXP, instead of validate if the values are different/not equals, in this way we could cover the value is going up by one (or the number we are expecting depending on the interactions we did) and that it is changing. :)

What do you think?

cc: @christianstokes

Reply:
Similar to what @christianstokes suggested here https://github.com/alexcandidos/liferay-portal/pull/493
 the only difference is that here we are validating the Page Views number instead of Individuals.